lib/active_model/serializable_resource.rb

```diff
@@ -30,7 +38,11 @@ def serialization_scope_name=(scope_name)
     end
 
     def adapter
-      @adapter ||= ActiveModel::Serializer::Adapter.create(serializer_instance, adapter_opts)
+      @adapter ||=
+        begin
```

Comment:
`begin/end` here is just a way to memoize the result of multiple lines. There's nothing special about it.  It's pretty standard Ruby doesn't need any special treatment.